Cargo.toml

```diff
@@ -14,6 +14,7 @@ readme = "README.md"
 anyhow = "1.0.91"
 clap = { version = "4.5.20", features = ["derive", "string"] }
 codespan-reporting = "0.11.1"
+once_cell = "1.19"
```

Comment:
This can be achieved using `std::sync::LazyLock` without adding a new dependency.